internal/multiparty/manager.go

```diff
@@ -155,7 +155,23 @@ func (mm *multipartyManager) configureContractCommon(ctx context.Context, migrat
 		}
 	}
 
-	subID, err := mm.blockchain.AddFireflySubscription(ctx, mm.namespace, current)
+	// For the case that we're establishing a listener from "latest" we obtain the protocol ID
+	// of the latest event confirmed from the blockchain for a given subscription.
+	// This protocolID should be parsed and used by the blockchain plugin if SubOptsFirstEventNewest
+	// is passed through, and the listener does not exist.
+	fb := database.BlockchainEventQueryFactory.NewFilter(ctx).Sort("-protocolid").Limit(1)
+	latestEvents, _, err := mm.database.GetBlockchainEvents(ctx, mm.namespace.Name, fb.Eq(
+		"listener", nil,
```

Comment:
Why are we not using the multiparty listener ID here? We don't have access to it?

Reply:
Specifically the thing that distinguishes the `blockchainevents` for the multi-party `BatchPin` events, is the lack of a `listener` reference (there isn't one in the FF model for the `BatchPin` case).

Reply:
Ah makes sense